regions/shapes/annulus.py

```diff
@@ -34,6 +36,8 @@ class CircleAnnulusPixelRegion(CompoundPixelRegion):
     """
 
     def __init__(self, center, inner_radius, outer_radius, meta=None, visual=None):
+        if inner_radius > outer_radius:
+            warnings.warn('Outer radius should be larger than inner radius.', AstropyUserWarning)
```

Comment:
this is better as an Exception than a warning; we are defining an annulus as r2>r1